libnopegl/src/node_animkeyframe.c

```diff
@@ -319,6 +323,175 @@ static easing_type back_derivative(easing_type t, easing_type s)
 DECLARE_EASINGS(back,            , back_func(x, PARAM(0, 1.70158)), TRANSFORM)
 DECLARE_EASINGS(back, _derivative, back_derivative(x, PARAM(0, 1.70158)), DERIVATIVE)
 
+/* Cubic Bezier */
+
+static int double_close_to(double value, double ref, double eps)
+{
+    return fabs(value - ref) < eps;
```

Comment:
`ref` and `eps` are always the same in your calls, so it could probably be called `close_to_zero` and take only one parameter